blockchain/utxobackend.go

```diff
@@ -229,6 +296,213 @@ func (l *LevelDbUtxoBackend) FetchStats() (*UtxoStats, error) {
 	return stats, nil
 }
 
+// dbPutUtxoDatabaseInfo uses an existing database transaction to store the
+// database information.
+func (f *LevelDbUtxoBackend) dbPutUtxoDatabaseInfo(dbTx database.Tx,
```

Comment:
Perhaps call it `dbPutUtxoBackendInfo` to match the rest?

Reply:
Agreed, updating the name of this function.